fv/etcd_restart_test.go

```diff
@@ -51,16 +51,7 @@ var _ = Context("etcd connection interruption", func() {
 
 	BeforeEach(func() {
 		felixes, etcd, client = infrastructure.StartNNodeEtcdTopology(2, infrastructure.DefaultTopologyOptions())
-
-		// Install a default profile that allows all ingress and egress, in the absence of any Policy.
-		defaultProfile := api.NewProfile()
-		defaultProfile.Name = "default"
-		defaultProfile.Spec.LabelsToApply = map[string]string{"default": ""}
-		defaultProfile.Spec.Egress = []api.Rule{{Action: api.Allow}}
-		defaultProfile.Spec.Ingress = []api.Rule{{Action: api.Allow}}
-		_, err := client.Profiles().Create(utils.Ctx, defaultProfile, utils.NoOptions)
-		Expect(err).NotTo(HaveOccurred())
-
+		infrastructure.CreateDefaultProfile(client, "default", map[string]string{"default": ""}, "")
```

Comment:
In the old code here there was no Source Selector, but CreateDefaultProfile will specify a Source Selector of `""`.  Is that equivalent?

Reply:
I've added a check in CreateDefaultProfile() to only assign a source selector if the given string's length is not zero.

Reply:
Thanks, that sounds good.